README.md

```diff
@@ -1,2 +1,54 @@
-# comp110-worksheet-B
+# comp110-worksheet-B (SCOTT CALVERT)
 Base repository for COMP110 worksheet B
+
+![pict](https://github.com/Mindengine76/comp110-worksheet-B/blob/master/WIN_20171019_19_32_18_Pro.jpg)
```

Comment:
Flowchart is a good attempt, but a little difficult to follow -- for example where is the start node? Also the blocks labelled "If", "True", "False" are confusing -- stick to the standard flowchart decision node (diamond shape).

Your flowchart was also a little hard to read due to the quality of the photo and the small writing in places -- drawing diagrams by hand is fine but make sure they are legible. Note that most of the FindMe printers on campus have a scan-to-email function.